com.obsproject.Studio.Plugin.obs_kinect.yaml

```diff
@@ -0,0 +1,83 @@
+id: com.obsproject.Studio.Plugin.obs_kinect
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/obs_kinect
+  prepend-pkg-config-path: /app/plugins/obs_kinect/lib/pkgconfig #needed for libfreenect2
+cleanup:
+  - /lib/obs-plugins/*.a
+modules:
+  - shared-modules/libusb/libusb.json
+
+#  - name: glfw
+#    buildsystem: cmake
+#    config-opts:
+#      - -DBUILD_SHARED_LIBS=ON
+#    sources:
+#      - type: archive
+#        url: https://github.com/glfw/glfw/releases/download/3.3.8/glfw-3.3.8.zip
+#        sha256: 4d025083cc4a3dd1f91ab9b9ba4f5807193823e565a5bcf4be202669d9911ea6
+
+#  - name: libjpeg-turbo
+#    buildsystem: cmake
+#    config-opts:
+#      - -DCMAKE_INSTALL_LIBDIR=/app/plugins/obs_kinect/lib
+#    sources:
+#      - type: archive
+#        url: https://github.com/libjpeg-turbo/libjpeg-turbo/archive/refs/tags/2.1.4.tar.gz
+#        sha256: a78b05c0d8427a90eb5b4eb08af25309770c8379592bb0b8a863373128e6143f
+
+#  - name: libk4a
+#    buildsystem: cmake-ninja
+#    sources:
+#      - type: archive
+#        url: https://github.com/microsoft/Azure-Kinect-Sensor-SDK/archive/refs/tags/v1.4.1.tar.gz
+#        sha256: 73106554449c64aff6b068078f0eada50c4474e99945b5ceb6ea4aab9a68457f
+
+  - name: libfreenect
+    buildsystem: cmake-ninja
+    cleanup:
+      - /lib/pkgconfig
+      - /lib/fakenect
+      - /include
+      - /bin
+    sources:
+      - type: git
+        url: https://github.com/SirLynix/libfreenect.git
+        commit: 5e2cd8e18877fbe7ef4c863161e90873c06a9180
+
+  - name: libfreenect2
+    buildsystem: cmake-ninja
+    cleanup:
+      - /lib/pkgconfig
+      - /lib/cmake
+      - /include
+    sources:
+      - type: archive
+        url: https://github.com/OpenKinect/libfreenect2/archive/refs/tags/v0.2.1.tar.gz
+        sha256: c09e52c97b0e90335f4762ed5363293ad0fc1ea0064e578b879e7d15cc3559df
+
+  - name: obs-kinect
+    buildsystem: simple
+    build-commands:
+      - install -D 64bit/* ${FLATPAK_DEST}/lib/
+      - mkdir -p ${FLATPAK_DEST}/share/obs/obs-plugins/
+      - mv obs-plugins/obs-kinect/ ${FLATPAK_DEST}/share/obs/obs-plugins/
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
+
+      - mkdir ${FLATPAK_DEST}/lib/obs-plugins # obs flatpak only merges /lib/obs-plugins/
+      - cp -a ${FLATPAK_DEST}/lib/{libobs*,libfreenect*,obs-kinect*} ${FLATPAK_DEST}/lib/obs-plugins/
+      - rm ${FLATPAK_DEST}/lib/{libobs*,libfreenect*,obs-kinect*}
+    sources:
+      - type: archive
+        url: https://github.com/SirLynix/obs-kinect/releases/download/v1.0-rc2/linux-1.0rc2-x64-releasedbg.zip
```

Comment:
this will have the debug symbols (guessing from the file name), and given this is an extension, the symbols are not extracted into a debug package (it's a known issue).

Reply:
this one has `releasedbg` in the name.

building from source, obviously would not raise this question

Reply:
I tried adding a `release` job in the [xmake github ci](https://github.com/o-kotb/obs-kinect/blob/master/.github/workflows/linux-build.yml), and it ended up with a [similar file size](https://github.com/o-kotb/obs-kinect/actions/runs/3904921068) as the `debug` one. However according to [xmake doc](https://xmake.io/#/manual/custom_rule), release and releasedbg are exactly the same with the only difference is those debug symbols are hidden. So if we ignore them, there won't be issues :)

> building from source, obviously would not raise this question

Don't worry sir, I AM building from source as you can see [here](https://github.com/SirLynix/obs-kinect/blob/master/.github/workflows/linux-build.yml). We can think of github ci as an extension to flathub's build system. So still reproducible and everything :). If there is some "convertgithubci2flatpakmanifest" tool out there I wouldn't mind using it, though.